test/Mono.Linker.Tests.Cases/Reflection/ExpressionPropertyString.cs

```diff
@@ -32,6 +35,58 @@ class Foo
 			[Kept]
 			[KeptBackingField]
 			private int TestName1 { [Kept] get; }
+
+			[Kept]
+			private string UnknownString ()
+			{
+				return "unknownstring";
+			}
+
+			[Kept]
+			public void Branch_SystemTypeValueNode_KnownStringValue_NonStatic ()
+			{
+				var expr = Expression.Property (Expression.Parameter (typeof (int), "somename"), typeof (Foo), "TestName1");
```

Comment:
Same comment as above - have a test for multiple type values and multiple name values.
And also the uniqueness problem with validation.